### ISSUE TITLE
Box Monitor in 6.4.62 is weird with my iStickTC100W 

### ISSUE THREAD

**Issue by smok-e** [TBXin/NFirmwareEditor#149]
The Box Monitor seems a bit confused.
It shows a Temperature of 36°C - 40°C (red line) when I take a puff, and the TC is set to 240°C (brown line)

and when I rescale the GUI of the box monitor (make it bigger ) the coloured lines go over the boxes.

maybe it is only my iStick and PC-usb connection, that is working weird.
But everything else is working fine.

**Comment by TBXin** [comment]
Can you attach a screenshot of this behavior?

**Comment by smok-e** [comment]
![box monitor-1](https://cloud.githubusercontent.com/assets/25061474/22390362/c6d4e792-e4ea-11e6-8ab0-85b6b2adcabf.PNG)
this is the screen with the blurred lines


and now the qweird temperature:
![temperature-puff-1](https://cloud.githubusercontent.com/assets/25061474/22390417/2a010ff8-e4eb-11e6-8dd1-1b6d7ca628ff.PNG)

I hope this explains what I meant

**Comment by TBXin** [comment]
@smok-e test, please, [this build](https://ci.appveyor.com/project/TBXin/nfirmwareeditor/build/6.4.64/artifacts).

**Comment by BigBadBlackWolf** [comment]
@ReikoKitsune look last pic "NFE Toolbox - Device Monitor" at this thread
from Stephan there are wrong words in Germany Translation from DE.Ipack.txt.

// Device Monitor Window
...
Toolbox.DeviceMonitor.OutputCurrentLabel=Ausgangsspannung:
Toolbox.DeviceMonitor.OutputVoltageLabel=Ausgansvolt:

OutputCurrent is Current not Voltage, also the correct word in German is
"Ausgangsstrom:"
OutputVoltage is Voltage, also the correct word in German is
"Ausgangsspannung:"



Toolbox.DeviceMonitor.RecordButton=Aufnehmen...
Toolbox.DeviceMonitor.RecordStopButton=Stop Aufnahme

Grammatic Correction
RecordButton=Start Aufnahme...
RecordStopButton=Stopp Aufnahme


Am 28.01.2017 11:18 schrieb "ReikoKitsune" <notifications@github.com>:

**Comment by smok-e** [comment]
I tested with the 6.4.65

the blurred lines are gone, but the recording is still weird:

![box monitor-2](https://cloud.githubusercontent.com/assets/25061474/22397235/8ae97ad0-e56d-11e6-920d-294c5140e662.PNG)

**Comment by TBXin** [comment]
@smok-e your coil material?

**Comment by smok-e** [comment]
Sorry, I think it was my mistake.
I think you're right, I don't remember the material I put into the atomizer,
but I changed the atomizer, and now the recording is working ok with SS304.

So maybe it would be helpfull, if there could be an error message when someone does 
soemthing like I tried. 
Maybe like: "Which material is the coil? Kanthal shows wrong values in TCR mode" or something like this. But maybe it is too complicated to programm it into the software?
